pr_testing/test_multiple_prs.sh

```diff
@@ -786,6 +787,27 @@ if ! $CMSDIST_ONLY ; then # If a CMSSW specific PR was specified #
     exit 0
   fi
 
+  if [[ $PRODUCTION_RELEASE == "true" && ${PULL_REQUEST} == *"/cmssw#"* ]]; then
+    pushd ${CMSSW_BASE}
+      mv src src.tmp
+      mkdir src && cd src
+      DSIZE=0
+      THRDS=""
+      git cms-init --upstream-only && git checkout -b codechecks $CMSSW_IB
+      git repack -h 2>&1 | grep '\-\-threads' && THRDS="--threads ${NCPU}" || true
+      git repack -a -d ${THRDS}
+      git repack -a -d ${THRDS}
+      OSIZE=$(du -sk .git/objects/pack | sed 's|\s.*||')
+      git cms-merge-topic --debug --ssh -u ${CMSSW_ORG}:${PR_NUMBER}
+      git repack -d ${THRDS}
+      NSIZE=$(du -sk .git/objects/pack | sed 's|\s.*||')
+      let DSIZE=${NSIZE}-${OSIZE} || DSIZE=0
+      if [ $DSIZE -gt 0 ]; then echo -e "\n* - This PR adds an extra ${DSIZE}KB to repository" > ${RESULTS_DIR}/16-git-repo-size-report.res; fi
```

Comment:
@iarspider , please change this to 
```
if [ $DSIZE -gt 0 ]; then echo "**Size**: This PR adds an extra ${DSIZE}KB to repository" > ${RESULTS_DIR}/09-git-repo-size-report.res; fi
```

this will return the summary as https://cmssdt.cern.ch/SDT/cgi-bin/get_pr_results/jenkins-artifacts/pull-request-integration/PR-7b41ad/40198/pr-result

#############################

**Failed Tests**: RelVals 
**Size**: This PR adds an extra 12KB to repository
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-7b41ad/40198/summary.html
**COMMIT**: 015d3ae04237944ad58e338ff7e259160cf5a536
**CMSSW**: CMSSW_14_1_X_2024-07-02-2300/el8_amd64_gcc12
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/45320/40198/install.sh` to create a dev area with all the needed externals and cmssw changes.


* **DAS Queries**: The DAS query tests failed, see the summary page for details.


## RelVals

- [29634.911](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-7b41ad/40198/runTheMatrix-results/29634.911_TTbar_14TeV+2026D110_DD4hep)```A fatal system signal has occurred: segmentation violation```